Metro-Records/la-metro-councilmatic
# Refactor uncategorized templates

## Overview

This continues the template upgrade to bootstrap 5, this time for the remaining uncategorized templates. These are mostly small templates with the exception of `legislation.html`.

Closes #999
Connects #969

### Notes

The templates that have been changed are:
 - 500
 - archive_search
 - contact
 - metro_login
 - legislation

## Testing Instructions
- Check the remaining templates against their live versions on desktop and mobile window sizes
- Confirm that
  - they looks presentable/similar to the original
  - the functionality still works (the login process, related links for legislations, archive, search, etc)
